moler/config/__init__.py

```diff
@@ -85,21 +87,18 @@ def load_config(config=None, from_env_var=None, config_type='yaml'):
         add_devices_only = True
         loaded_config.append(config)
 
-    if (config_type != 'dict') and (config_type != 'yaml'):  # no other format supported yet
-        raise WrongUsage("Unsupported config_type: '{}'. Allowed are: 'dict' or 'yaml'.".format(config_type))
+    if config is None and from_env_var is None:
+        raise WrongUsage("Provide either 'config' or 'from_env_var' parameter (none given).")
+    elif not from_env_var and (not isinstance(config, six.string_types) and not isinstance(config, dict)):  # no other format supported yet
```

Comment:
wrong check - what if `from_env_var="AAA"` and `config=[1,2,3]`? It won't be detected.
condition should be:
`elif config and (not isinstance(config, six.string_types) and not isinstance(config, dict)):`

Reply:
0fef3d93c7b7e81dfec3d4ae2e826dbe5d57d139